pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -530,20 +530,18 @@ func updateObjectsLabels(crLables map[string]string, objs []*unstructured.Unstru
 		// Label objects with version of the operator they were created by
 		labels[cnaov1.SchemeGroupVersion.Group+"/version"] = operatorVersionLabel
 		labels[names.PROMETHEUS_LABEL_KEY] = names.PROMETHEUS_LABEL_VALUE
-		err = updateObjectTemplateLabels(obj, labels, []string{names.PROMETHEUS_LABEL_KEY})
-		if err != nil {
-			return err
-		}
-
-		appLabelKeys := []string{names.COMPONENT_LABEL_KEY, names.PART_OF_LABEL_KEY, names.VERSION_LABEL_KEY, names.MANAGED_BY_LABEL_KEY}
+		labels[names.MANAGED_BY_LABEL_KEY] = names.MANAGED_BY_LABEL_DEFAULT_VALUE
 
+		appLabelKeys := []string{names.COMPONENT_LABEL_KEY, names.PART_OF_LABEL_KEY, names.VERSION_LABEL_KEY}
 		labels = updateLabelsFromCR(labels, crLables, appLabelKeys)
 		if err != nil {
 			return err
 		}
 		obj.SetLabels(labels)
 
-		err = updateObjectTemplateLabels(obj, labels, appLabelKeys)
+		var templateLabelKeys []string
+		templateLabelKeys = append(appLabelKeys, names.PROMETHEUS_LABEL_KEY, names.MANAGED_BY_LABEL_KEY)
```

Comment:
please consider doing those two in one line []{appLabelKeys..., names.PROMETHEUS_LABEL_KEY, names.MANAGED_BY_LABEL_KEY} (if it works)

or even just 
```
		templateLabelKeys := append(appLabelKeys, names.PROMETHEUS_LABEL_KEY, names.MANAGED_BY_LABEL_KEY)
```